source/_components/worldclock.markdown

```diff
@@ -22,6 +22,7 @@ To enable this sensor in your installation, add the following to your `configura
 sensor:
   - platform: worldclock
     time_zone: America/New_York
+    time_format: "%-I:%M %p"
```

Comment:
Only required variables in the first example

Reply:
Thanks for the info! Made the changes.